packages/composites/multi-address-list/src/config/index.ts

```diff
@@ -0,0 +1,51 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  ANCHORAGE_ADAPTER_URL: {
+    description: 'URL of Anchorage EA',
+    type: 'string',
+    required: true,
+  },
+  BITGO_ADAPTER_URL: {
+    description: 'URL of Bitgo EA',
+    type: 'string',
+    required: true,
+  },
+  COINBASE_PRIME_ADAPTER_URL: {
+    description: 'URL of Coinbase Prime EA',
+    type: 'string',
+    required: true,
+  },
+  SCHEDULER_HOUR: {
+    description: 'Hour to run scheduler [0-23]',
+    type: 'number',
+    default: 17,
+  },
+  SCHEDULER_MINUTES: {
+    description: 'Minute to run scheduler [0-59]',
+    type: 'number',
+    default: 1,
+  },
+  SCHEDULER_TIMEZONE: {
+    description: 'Timezone to run scheduler',
+    type: 'string',
+    default: 'America/New_York',
```

Comment:
😬

Reply:
can we make sure to get sign off for daylight savings at least? there will be an hour gap/overlap that day and i'd rather call it out super early